gitpod-io/gitpod
# [server] infer extensions and add a comment

This commit adds vscode extensions to the inferred `.gitpod.yml` and adds a comment to the top, that links to the documentation.

follow up to: https://github.com/gitpod-io/gitpod/pull/7383

## How to test
Start workspaces on repositories without `.gitpod.yml` and check the generated `.gitpod.yml`.

## Release Notes
```release-note
Also propose vscode extensions for yet unconfigured repositories.
```
